packages/slate-react/src/plugins/before.js

```diff
@@ -103,8 +103,14 @@ function BeforePlugin() {
     // If the value's schema isn't the editor's schema, update it. This can
     // happen on the initialization of the editor, or if the schema changes.
     if (value.schema != editor.schema) {
+      // When change on initialization, don't save change into history, making
+      // this schema as default and history clean after initialization ends.
+      const isInit = (
+        value.history.undos.size === 0 &&
+        value.history.redos.size === 0
+      )
       change
-        .setValue({ schema: editor.schema })
+        .setValue({ schema: editor.schema }, { save: !isInit })
```

Comment:
Actually, I think we could just always not save for this one? Since no matter what it's going to continue to get set even if the user does undo it?

Reply:
Fixed!